pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -579,7 +581,8 @@ func applyHTTPRouteDestination(
 	} else {
 		action.ClusterSpecifier = &route.RouteAction_WeightedClusters{
 			WeightedClusters: &route.WeightedCluster{
```

Comment:
we have more usage in buildOutboundNetworkFiltersWithWeightedClusters I think?

Reply:
TcpProxy_WeightedCluster has no total_weight field

Reply:
That's very weird, and i filed a issue to make this optional https://github.com/envoyproxy/envoy/issues/21845